lib/Net/Async/Blockchain.pm

```diff
@@ -62,6 +65,9 @@ must be included and removed here.
 =item * C<subscription_msg_timeout> Subscription interval between messages timeout
 =item * C<currency_symbol> Currency symbol
 =item * C<base_block_number> Block number where the subscription must apply the recursive search from
+=item * C<redis_host> Redis host. (optional, default: '127.0.0.1')
+=item * C<redis_port> Redis port. (optional, default: '6379')
+=item * C<redis_auth> Redis password. (optional, default: '')
```

Comment:
Technically the default is undef, I think?

Reply:
Yes recently updated the `redis_auth` default value , forget to update POD thanks..